### ISSUE TITLE
Duplication of @removeNullVariables directive in introspection schema

### ISSUE THREAD

**Issue by HXavS** [wundergraph/wundergraph#985]
### Bug description

adding single api with `introspect.postgresql({...` is duping @removeNullVariables and causes error on 3rd party graphql introspection.
#477 

### How to reproduce

1. create new nextjs-prisma-postgres example 
2. Add Database URL
3. Run 'yarn wunderctl up --logs'
4. See error when trying to introspect local with [3rd party Graphql UI](https://studio.apollographql.com/sandbox/explorer)

### Expected behavior

Generate Valid Schema that doesn't cause errors

### WunderGraph information

    "@wundergraph/nextjs": "^0.10.29",
    "@wundergraph/sdk": "^0.158.1",

### Environment & setup

- OS:  Mac OS - M1 Pro
- Go version: go1.20.2 darwin/arm64
- Database: PostgreSQL 15
- Node.js version: 18.X

### WunderCtl Version

Version: 0.152.1

**Comment by fiam** [comment]
@HXavS The `@removeNullVariables` seems to be correct in our tests. Could you please let us know which URL are your endpoint as well as providing an screenshot of the error that you're receiving? Thanks!

**Comment by HXavS** [comment]
The schema file generated looks correct. But external Graphql explorers, https://studio.apollographql.com/sandbox/explorer, run introspection query by when running my network response is show how duplicating `removeNullVariables`. 

I get the error when running `yarn wundergraph` and trying to connect locally at `http://localhost:9991/graphql`


```
    query IntrospectionQuery {
      __schema {
        
        queryType { name }
        mutationType { name }
        subscriptionType { name }
        types {
          ...FullType
        }
        directives {
          name
          description
          
          locations
          args {
            ...InputValue
          }
        }
      }
    }

    fragment FullType on __Type {
      kind
      name
      description
      
      fields(includeDeprecated: true) {
        name
        description
        args {
          ...InputValue
        }
        type {
          ...TypeRef
        }
        isDeprecated
        deprecationReason
      }
      inputFields {
        ...InputValue
      }
      interfaces {
        ...TypeRef
      }
      enumValues(includeDeprecated: true) {
        name
        description
        isDeprecated
        deprecationReason
      }
      possibleTypes {
        ...TypeRef
      }
    }

    fragment InputValue on __InputValue {
      name
      description
      type { ...TypeRef }
      defaultValue
      
      
    }

    fragment TypeRef on __Type {
      kind
      name
      ofType {
        kind
        name
        ofType {
          kind
          name
          ofType {
            kind
            name
            ofType {
              kind
              name
              ofType {
                kind
                name
                ofType {
                  kind
                  name
                  ofType {
                    kind
                    name
                  }
                }
              }
            }
          }
        }
      }
    }
  ```

<img width="884" alt="Screenshot 2023-06-02 at 10 39 26 AM" src="https://github.com/wundergraph/wundergraph/assets/46010924/a6311ba5-a44a-42be-ac6d-2dd07d52a70b">

**Comment by konsti** [comment]
I have the same issue

**Comment by Aenimus** [comment]
Fixed as of `0.174.5`. Thank you for the reports @konsti @HXavS!
